### ISSUE TITLE
Weapon Skills Update Coming Soon

### ISSUE THREAD

**Issue by Siryphas** [Stexinator/TheWitcherTRPG#122]
Before anyone makes an issue of this, I _will_ update the Compendium, but I use Forge and need to wait until they release v.12 before I can do any more work on it.

**Comment by AlexanderTheGr3at** [comment]
You can update your forge server in the configuration. Just scroll down to Foundry VTT Server Version and select v12
![Screenshot_20240604_120916_Firefox](https://github.com/Stexinator/TheWitcherTRPG/assets/59134326/1fc1c62a-fadf-47b6-8cb0-5f32c4ba9ca9)

**Comment by Siryphas** [comment]
Yes, I know _how_, thank you. It just hadn't been released at the time of writing and we hadn't been given a release date. Naturally, as soon as I make a post, it would drop the next day.

**Comment by AlexanderTheGr3at** [comment]
> Yes, I know _how_, thank you. It just hadn't been released at the time of writing and we hadn't been given a release date.

Sorry, was just trying to help.

> Naturally, as soon as I make a post, it would drop the next day.

Ain't that how it always goes lol.

**Comment by Siryphas** [comment]
Work on the Compendium will begin, but I don't have an update ETA just yet 🙇‍♂️
